### PR TITLE
style(settings): add styles in settings menu form

### PR DESCRIPTION
**Pull request by prasanth94**
## Why are the changes necessary?

- To fix issue - This PR closes issue #360 

## What does this pull request cover?

- Add styles to form in settings menu, used tailwind classes to style the component.
- Tailwind has been added with the following configs
   1. 'height' property is added to transitionProperty
   2. 'boxShadow' property is added to the 'active' pseudo class.
   3. 'primary'  property for color  has been added to access application's primary color from tailwind classes.

## Demo

jina-settings

### CODE REVIEW

File: src/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -26,72 +25,67 @@ function SettingsCard() {
...
+    <div className=" bg-white  rounded-md">
+      <h6 className="p-4 border-b text-gray-600">Connection Preferences</h6>
+      <div className="p-4">
+        <div className="grid grid-cols-2 gap-4 mb-4">{renderBaseOptions}</div>
+        <div className="flex mb-4 justify-between items-center">
+          <ToggleButton
+            isExpanded={isToggleButtonExpanded}
+            onToggleButtonClick={handleToggleButtonClick}
+          />
+          <button
```
Comment (prasanth94) [inline src/components/Settings/Settings.tsx:68]: The button component inside common folder is not used since its prop variant="primary" is not exactly rendering the primary color of the app. Hence used button element with tailwind classes. In future we can change to common button component when the variant="primary" is working as expected.

File: tailwind.config.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
+const colors = require("tailwindcss/colors")
+
 module.exports = {
-  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
```
Comment (prasanth94) [inline tailwind.config.js:2]: This change is not related to this PR, but it fixes the formatting style to double quotes.

File: src/components/Settings/FormItem.tsx [hunk 3]
```diff
@@ -6,21 +6,25 @@ type Props = {
   label: string
   value: SettingName
   placeholder: string
-  onChange: (e: ChangeEvent<HTMLInputElement>) => void
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void
```
Comment (prasanth94) [inline src/components/Settings/FormItem.tsx:9]: This change is not related to this PR. Refactor to better variable name

**Review by Roshanjossey — APPROVED** [review]

**Review by BastinJafari — APPROVED** [review]
